autogpt_platform/backend/backend/blocks/example/example.py

```diff
@@ -0,0 +1,143 @@
+            description="The number of greetings in the input."
+        )
+        error: str = SchemaField(description="The error from the example block")
+
+    def __init__(self):
+        super().__init__(
+            # The unique identifier for the block, this value will be persisted in the DB.
+            # It should be unique and constant across the application run.
+            # Use the UUID format for the ID.
+            id="380694d5-3b2e-4130-bced-b43752b70de9",
+            # The description of the block, explaining what the block does.
+            description="The example block",
+            # The list of contributors who contributed to the block.
+            # Each contributor is represented by a ContributorDetails object.
+            contributors=[ContributorDetails(name="Craig Swift")],
+            # The set of categories that the block belongs to.
+            # Each category is an instance of BlockCategory Enum.
+            categories={BlockCategory.BASIC},
+            # The schema, defined as a Pydantic model, for the input data.
+            input_schema=ExampleBlock.Input,
+            # The schema, defined as a Pydantic model, for the output data.
+            output_schema=ExampleBlock.Output,
+            # The list or single sample input data for the block, for testing.
+            # This is an instance of the Input schema with sample values.
+            test_input={
+                "name": "Craig",
+                "greetings": ["Hello", "Hi", "Hey"],
+                "is_funny": True,
+                "credentials": TEST_CREDENTIALS_INPUT,
+            },
+            # The list or single expected output if the test_input is run.
+            # Each output is a tuple of (output_name, output_data).
+            test_output=[
+                ("response", {"message": "Hello, world!"}),
+                ("response", {"message": "Hello, world!"}),  # We mock the function
+                ("response", {"message": "Hello, world!"}),  # We mock the function
+                (
+                    "all_responses",
+                    [
+                        {"message": "Hello, world!"},
+                        {"message": "Hello, world!"},
+                        {"message": "Hello, world!"},
+                    ],
+                ),
+                ("greeting_count", 3),
+            ],
+            # Function names on the block implementation to mock on test run.
+            # Each mock is a dictionary with function names as keys and mock implementations as values.
+            test_mock={
+                "my_function_that_can_be_mocked": lambda *args, **kwargs: "Hello, world!"
+            },
+            # The credentials required for testing the block.
+            # This is an instance of APIKeyCredentials with sample values.
+            test_credentials=TEST_CREDENTIALS,
+            # The type of the block, which is an instance of BlockType Enum.
+            block_type=BlockType.STANDARD,
```

Comment:
Won't this block be marked as disabled? If not it will show up on the block list.

Reply:
I do not follow - are you saying this block should be marked as disabled? or that using `BlockType.STANDARD` marks the block as disabled?

Reply:
block_type Standard is normal and should not need specified

Reply:
I've removed the block_type Standard